projects/ngx-maplibre-gl/src/lib/popup/popup.component.ts

```diff
@@ -41,67 +43,70 @@ import { MarkerComponent } from '../marker/marker.component';
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
 })
-export class PopupComponent implements OnChanges, OnDestroy, OnInit {
-  /** Init input */
-  @Input() closeButton?: PopupOptions['closeButton'];
-  /** Init input */
-  @Input() closeOnClick?: PopupOptions['closeOnClick'];
-  /** Init input */
-  @Input() closeOnMove?: PopupOptions['closeOnMove'];
-  /** Init input */
-  @Input() focusAfterOpen?: PopupOptions['focusAfterOpen'];
-  /** Init input */
-  @Input() anchor?: PopupOptions['anchor'];
-  /** Init input */
-  @Input() className?: PopupOptions['className'];
+export class PopupComponent implements OnChanges, OnInit {
+  /** Init injection */
+  private readonly destroyRef = inject(DestroyRef);
+  private readonly mapService = inject(MapService);
+
   /** Init input */
-  @Input() maxWidth?: PopupOptions['maxWidth'];
+  readonly closeButton = input<PopupOptions['closeButton']>();
+  readonly closeOnClick = input<PopupOptions['closeOnClick']>();
```

Comment:
There are missing comments here that were before about init input and dynamic input.
Can you go over the files and add the init/dynamic input comment for every input parameter please?